tesseract_motion_planners/CMakeLists.txt

```diff
@@ -87,3 +87,69 @@ endif()
 
 # Package configuration
 configure_package(NAMESPACE tesseract)
+
+if(TESSERACT_PACKAGE)
+  set(LINUX_DEPENDS
+      "libconsole-bridge-dev"
+      "libeigen3-dev"
+      "libboost-dev"
+      "${TESSERACT_PACKAGE_PREFIX}tesseract-common"
+      "${TESSERACT_PACKAGE_PREFIX}tesseract-command-language"
+      "${TESSERACT_PACKAGE_PREFIX}tesseract-environment")
+  set(WINDOWS_DEPENDS
+      "console-bridge"
+      "Eigen3"
+      "boost"
+      "${TESSERACT_PACKAGE_PREFIX}tesseract-common"
+      "${TESSERACT_PACKAGE_PREFIX}tesseract-command-language"
+      "${TESSERACT_PACKAGE_PREFIX}tesseract-environment")
+  if(TESSERACT_BUILD_OMPL)
+    list(APPEND LINUX_DEPENDS "libompl-dev")
+    list(APPEND WINDOW_DEPENDS "ompl")
+  endif()
+  if(TESSERACT_BUILD_DESCARTES)
+    list(
+      APPEND
+      LINUX_DEPENDS
+      "descartes-light"
+      "${TESSERACT_PACKAGE_PREFIX}tesseract-collision")
+    list(
+      APPEND
+      WINDOW_DEPENDS
+      "descartes_light"
+      "${TESSERACT_PACKAGE_PREFIX}tesseract-collision")
+  endif()
+  if(TESSERACT_BUILD_TRAJOPT)
+    list(
+      APPEND
+      LINUX_DEPENDS
+      "trajopt"
```

Comment:
Do you think we should prefix these with `${TRAJOPT_PACKAGE_PREFIX}` in the case someone builds these packages with a non-empty prefix (e.g. `ros-noetic-`)?

Reply:
This way you could have both installed if you wanted.